schemas/json/theme.json

```diff
@@ -2244,7 +2249,7 @@
 		"version": {
 			"description": "Version of theme.json to use.",
 			"type": "integer",
-			"enum": [ 2 ]
+			"enum": [ 3 ]
```

Comment:
I think we still need to support the older versions. Having only version 3 in the enum list produces a warning for default themes.

![CleanShot 2024-04-30 at 11 33 35](https://github.com/WordPress/gutenberg/assets/240569/bb37ea3b-17c6-4f6b-822b-c460fd7210a0)

Reply:
Or should we increase the default theme's theme.json version to 3?

Reply:
I mentioned the default theme as an example. Not everyone will upgrade `versions` immediately, and the older versions still work.

Reply:
Default themes, like all other themes, should be using the schema for the oldest version of WordPress that they intend to support so that incompatible styles and settings are properly flagged. If the version is updated to 3, then the theme will only work in 6.6 and up.